domain/src/main/java/no/difi/meldingsutveksling/arkivmelding/AvskrivningsmaateMapper.java

```diff
@@ -0,0 +1,36 @@
+package no.difi.meldingsutveksling.arkivmelding;
+
+import com.google.common.collect.HashBiMap;
+import lombok.extern.slf4j.Slf4j;
+import no.arkivverket.standarder.noark5.metadatakatalog.Avskrivningsmaate;
+
+@Slf4j
+public class AvskrivningsmaateMapper {
+
+    private static final HashBiMap<String, Avskrivningsmaate> mapper = HashBiMap.create();
+
+    static {
+        mapper.put("BU", Avskrivningsmaate.BESVART_MED_BREV);
+        // TODO: potential wrong mapping
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1135)